src/Components/SingleMovie/SingleMovie.js

```diff
@@ -8,17 +8,26 @@ const SingleMovie = (props) => {
   //   return null;
   // }
 
+  React.useEffect(() => {
+    document.body.addEventListener('keydown', props.closeOnEscapeKey)
+    return function cleanup() {
+      document.body.removeEventListener('keydown', props.closeOnEscapeKey)
```

Comment:
Let's make sure we talk to Robbie and understand for sure what this cleanup function is doing